controllers/account/ec2.go

```diff
@@ -803,7 +803,7 @@ func vCPUQuotaNeedsIncrease(client awsclient.Client, desiredQuota float64) (bool
 
 	// Regardless of errors, if we got the result for the actual quota,
 	// then compare it to the desired quota.
-	if result.Quota != nil {
+	if result.Quota != nil && result.Quota.Value != nil {
```

Comment:
This resulted from some early int test runs where I was occasionally seeing seg faults in AAO. Probably just a weird test scenario but I havent seen it since adding this and I cant see a reason not to have the additional check.